mage_ai/server/execution_manager.py

```diff
@@ -0,0 +1,41 @@
+from distutils.dir_util import copy_tree
+import os
+import shutil
+
+class PipelineExecution():
```

Comment:
Could you add some comments to this class since it's not easy to understand what it does without context?

Reply:
You can remove the `()` after class name if it doesn't have parent classes.